include/btllib/indexlr.hpp

```diff
@@ -147,6 +167,16 @@ class Indexlr
           const btllib::BloomFilter& bf1 = Indexlr::dummy_bf(),
           const btllib::BloomFilter& bf2 = Indexlr::dummy_bf());
 
+  Indexlr(std::string seqfile,
```

Comment:
I don't think doxygen will generate the documentation for this constructor. We'll need two separate sets of documentation comments for the constructors, one with `q` and one without.